spot_driver/spot_driver/spot_wrapper.py

```diff
@@ -464,7 +520,7 @@ def releaseEStop(self):
 
     def getLease(self):
         """Get a lease for the robot and keep the lease alive automatically."""
-        self._lease = self._lease_client.acquire()
+        self._lease = self._lease_client.take()
```

Comment:
Note this change.  This will take the lease even if something else has it, which made switching between ROS and tablet operation smoother for us.  If this is a problem, it can be a variable!